crates/runtime/src/dataconnector/clickhouse.rs

```diff
@@ -233,16 +233,14 @@ impl DataConnector for Clickhouse {
         &self,
         dataset: &Dataset,
     ) -> super::DataConnectorResult<Arc<dyn TableProvider>> {
-        Ok(Read::table_provider(
-            &self.clickhouse_factory,
-            dataset.path().into(),
-            dataset.schema(),
+        Ok(
+            Read::table_provider(&self.clickhouse_factory, dataset.path().into(), None)
```

Comment:
Should we update `ReadWrite::table_provider` since it looks like they all provide `schema: None`.

Reply:
Yeah I'll delete it all.